tools/vulnerability-scan/wss-scan.sh

```diff
@@ -47,33 +47,70 @@ basepath=$baseDirPath"/repos"
 
 echo "Cleaning up scan directories if already present"
 rm -rf $basepath
+echo "Cleaning up temp file that will affect scan"
+rm -rf /tmp/ws*
 
 mkdir -p $basepath
  
 # clone the desired Repos for scanning 
 for repo in $gitRepos
 do
   echo "Cloning repo "$gitBasePath$repo
-  git clone "$gitBasePath$repo".git $basepath"/"$repo
+  if [ ${repo} == "OpenSearch" ]
+  then
+    echo "Cloning "$repo" at branch 1.x"
+    git clone -b 1.x "$gitBasePath$repo".git ${basepath}/${repo}
+    rm -rf ${basepath}/${repo}/qa
```

Comment:
Remove QA is due to issues listed in https://github.com/opensearch-project/opensearch-build/pull/1411#issuecomment-1000945335.

@zelinh please add a comment in case we forget about this.

Reply:
We dont mention ES in the code, use ODFE.